app/main/posts/views/post-view-data.html

```diff
@@ -1,4 +1,11 @@
+<div class="mode-context init" embed-only="true" ng-controller="navigation as nav" dropdown>
+    <header class="mode-context-header" ng-class="{ 'has-logo' : nav.site.image_header }">
+      <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
+      <img ng-if="nav.site.image_header" ng-src="{{ nav.site.image_header }}" class="deployment-logo" />
+    </header>
```

Comment:
Above is ugly, but I couldn't find a way to add it somewhere else. Its just to get the deployment-header in data-view in embeds.